src/interfaces/document.interface.ts

```diff
@@ -5,7 +5,7 @@ export interface Document {
   user: number;
   users: number[];
   folder_id: number | null;
-  expires: number | null;
+  expires: string | null;
```

Comment:
Is this a new change? If so is this for v1 or v2 of the API or has this type always been a string and we just got it wrong initially?